NativeApp/src/components/Common/HeaderDays/index.js

```diff
@@ -13,29 +12,29 @@ const HeaderDays = (props) => {
     <View style={styles.headerContainer}>
       <View style={styles.holidayContainer}>
         <View style={styles.holidayText}>
-          <H1 type="bold">
-            {takenHolidays.length}
+          <H1 type="base">
```

Comment:
You might not need to pass base type if it's set as a default within the H1 component.